src/Sparrow/Platform/Win32/Win32ProcessMethods.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Sparrow.Platform.Win32
```

Comment:
Needs to be moved to Sparrow.Server

Reply:
Botched merge, that is not used. Removing